subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_manager.c

```diff
@@ -158,5 +161,9 @@ int bt_fast_pair_factory_reset(void)
 		return err;
 	}
 
+	if (fast_pair_info_cb && fast_pair_info_cb->account_key_list_changed) {
+		fast_pair_info_cb->account_key_list_changed(NULL);
```

Comment:
btw. The Account Key list would be actually unchanged if there were no Account Keys stored. Should we call the callback in such case?
Maybe we could move calling the callback to the `fp_storage_ak` - it seems responsibility of the module that handles  Account Key storage.